src/adapters/sqlite/index.js

```diff
@@ -64,13 +81,79 @@ type NativeBridgeType = {
   getLocal: (ConnectionTag, string) => Promise<?string>,
   setLocal: (ConnectionTag, string, string) => Promise<void>,
   removeLocal: (ConnectionTag, string) => Promise<void>,
+}>
+
+const dispatcherMethods = [
+  'initialize',
+  'setUpWithSchema',
+  'setUpWithMigrations',
+  'find',
+  'query',
+  'count',
+  'batch',
+  'batchJSON',
+  'getDeletedRecords',
+  'destroyDeletedRecords',
+  'unsafeResetDatabase',
+  'getLocal',
+  'setLocal',
+  'removeLocal',
+]
+
+type NativeBridgeType = {
+  // Async methods
+  ...NativeDispatcher,
+
+  // Synchronous methods
+  initializeSync?: (ConnectionTag, string, SchemaVersion) => SyncReturn<InitializeStatus>,
```

Comment:
I'm not a fan of `*Sync` naming scheme, it seems confusing especialy when we are talking about database that can `Synchronize` with server. Maybe something like `Synch` 🤷🏻‍♂️

Reply:
riiiight. I'll go with `*Synchronous`